Modules/arraymodule.c

```diff
@@ -1698,9 +1699,7 @@ array_array_fromunicode_impl(arrayobject *self, PyObject *ustr)
 
         Py_ssize_t res = PyUnicode_AsWideChar(
                 ustr, ((wchar_t *)self->ob_item) + old_size, ustr_length);
-        if (res < 0) {  // must not happen
-            return NULL;
-        }
+        assert(res == ustr_length);
```

Comment:
Sadly such code emits a "variable unused" in release mode, since the assertion is removed :-(

Reply:
This could have also been fixed by adding a `__attribute__((unused));` and keeping the assert. This is minor anyway, so I don't think you should need to bother now :P

Reply:
Is it portable?  Does MSVC support it?
The technique I know is `(void)res;`.
But I think this assert is not important enough to introduce such garbage code.

Reply:
MSVC does support function attributes but it does not use the same syntax as GCC and Clang. You would need to use the preprocessor to define it based on the compiler. It is probably not worth for this.